etc/encoding/opencast.properties

```diff
@@ -0,0 +1,243 @@
+  #{out.dir}/#{out.name}#{out.suffix}
+
+# Slide preview images as shown in the player
+profile.player-slides.http.name = slide preview image for engage
+profile.player-slides.http.input = visual
+profile.player-slides.http.output = image
+profile.player-slides.http.suffix = .#{time}.jpg
+profile.player-slides.http.ffmpeg.command = -ss #{time} -i #{in.video.path} \
+  -r 1 -frames:v 1 -filter:v yadif,scale=160:-1 \
+  #{out.dir}/#{out.name}#{out.suffix}
+
+# Cover image for search results
+profile.search-cover.http.name = cover image for engage
+profile.search-cover.http.input = visual
+profile.search-cover.http.output = image
+profile.search-cover.http.suffix = -search.jpg
+profile.search-cover.http.ffmpeg.command = -i #{in.video.path} -frames:v 1 \
+  -filter:v select=eq(n\\,1)+lt(t\\,#{time}),fps=1,reverse,scale=160:-2 \
+  #{out.dir}/#{out.name}#{out.suffix}
+
+profile.mp4-preview.http.name = preview video
+profile.mp4-preview.http.input = visual
+profile.mp4-preview.http.output = visual
+profile.mp4-preview.http.suffix = -preview.mp4
+profile.mp4-preview.http.ffmpeg.command = -i #{in.video.path} \
+  -filter:v yadif,scale=-2:360 \
+  -c:v libx264 -preset veryfast -crf 23 -profile:v baseline -pix_fmt yuv420p -tune film \
+  -c:a aac -strict -2 -ar 22050 -ab 64k \
+  -movflags faststart #{out.dir}/#{out.name}#{out.suffix}
+
+profile.mp4-preview.dual.http.name = preview video (picture-by-picture)
+profile.mp4-preview.dual.http.input = visual
+profile.mp4-preview.dual.http.output = visual
+profile.mp4-preview.dual.http.suffix = -preview-composite.mp4
+profile.mp4-preview.dual.http.ffmpeg.command = -i #{in.video.path} \
+  #{compositeCommand} \
+  -c:v libx264 -preset veryfast -crf 23 -profile:v baseline -pix_fmt yuv420p -tune film \
+  -c:a aac -strict -2 -ar 22050 -ab 64k \
+  -movflags faststart  \
+  #{out.dir}/#{out.name}#{out.suffix}
+
+# Distribution format definition
+profile.fast.http.name = fast processed mp4
+profile.fast.http.input = visual
+profile.fast.http.output = visual
+profile.fast.http.suffix = .mp4
+profile.fast.http.ffmpeg.command = -i #{in.video.path} \
+  -filter:v crop=in_w/2*2:in_h/2*2 \
+  -c:a aac -c:v libx264 -preset faster -g 30 -pix_fmt yuv420p \
+  #{out.dir}/#{out.name}#{out.suffix}
+
+# Cover image for atom and rss feeds
+profile.feed-cover.http.name = cover image for feeds
+profile.feed-cover.http.input = visual
+profile.feed-cover.http.output = image
+profile.feed-cover.http.suffix = -feed.jpg
+profile.feed-cover.http.ffmpeg.command = -ss #{time} -i #{in.video.path} \
+  -r 1 -frames:v 1 -filter:v yadif,scale=-1:54 \
+  #{out.dir}/#{out.name}#{out.suffix}
+
+# Image to video
+profile.image-movie.work.name = image to video
+profile.image-movie.work.input = image
+profile.image-movie.work.output = visual
+profile.image-movie.work.suffix = -image-video.mp4
+profile.image-movie.work.ffmpeg.command = -loop 1 -i #{in.video.path} \
+  -c:v libx264 -r 30 -t #{time} -pix_fmt yuv420p \
```

Comment:
This may result in:

```
[libx264 @ 0x509bac0] width not divisible by 2 (225x225)                                                                                                                                                                                      
Error initializing output stream 0:0 -- Error while opening encoder for output stream #0:0 - maybe incorrect parameters such as bit_rate, rate, width or height
```